packages/config/src/layer2s/common/index.ts

```diff
@@ -8,3 +8,10 @@ export * from './nuggets'
 export * from './operator'
 export * from './riskView'
 export * from './stateCorrectness'
+
+export const EMPTY_TECHNOLOGY_CHOICE = {
+  name: '–',
+  description: 'Not provided',
+  references: [],
+  risks: [],
+}
```

Comment:
The index file ideally shouldn't have any content outside of exports. If we need to create a new file just for this, fine, let's do it.

Reply:
I will move it to other file, let's not break the pattern from our codebase